components/wsdl-tools/wsdlvalidator/org.wso2.carbon.tools.wsdlvalidator/src/main/java/org/wso2/carbon/tools/wsdlvalidator/WsdlValidator.java

```diff
@@ -239,7 +239,7 @@ private URL loadXMLToFile(Document document) throws TransformerException, IOExce
             transformerFactory = TransformerFactory
                     .newInstance("com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl", null);
         } catch (NoSuchMethodError e) {
-            logger.log(Level.INFO, "TransformerFactory.newInstance(String, ClassLoader) method not found. " +
+            logger.info("TransformerFactory.newInstance(String, ClassLoader) method not found. " +
```

Comment:
This is logged as an info since it is not a warning or an error. One of the methods will anyway be invoked